src/geom/cell_hex27.C

```diff
@@ -97,6 +97,14 @@ bool Hex27::is_node_on_side(const unsigned int n,
   return false;
 }
 
+std::vector<unsigned> Hex27::nodes_on_side(const unsigned int s) const
+{
+  libmesh_assert_less (s, n_sides());
+  return std::vector<unsigned>(side_nodes_map[s],
+                               side_nodes_map[s]
+                               + sizeof(side_nodes_map[s]) / sizeof(side_nodes_map[s][0]));
+}
```

Comment:
The verbosity of the `sizeof`/`sizeof` calculation isn't ideal. Since the number of nodes on each side is also used in the definition of the `side_nodes_map` itself, it should probably be a `static` `const` member of the class for simplicity, but that could wait for a later PR.

Can you use curly-brace initialization to avoid explicitly repeating the type, i.e.
```
return {side_nodes_map[s], side_nodes_map[s] + 9};
```
I think that should compile...

Reply:
Yes I think a `static const` member `num_nodes_on_side` could be used in multiple places, and so I think a later PR would be good for that